extensions/clusters/clusters.go

```diff
@@ -454,19 +459,26 @@ func NewK3SRKE2ClusterConfig(clusterName, namespace string, clustersConfig *Clus
 		MachinePools:         machinePools,
 	}
 
+	agentEnvVars := []rkev1.EnvVar{}
+
 	spec := apisV1.ClusterSpec{
 		CloudCredentialSecretName:           cloudCredentialSecretName,
 		KubernetesVersion:                   clustersConfig.KubernetesVersion,
 		LocalClusterAuthEndpoint:            localClusterAuthEndpoint,
 		RKEConfig:                           rkeConfig,
 		ClusterAgentDeploymentCustomization: clusterAgentDeploymentCustomization,
 		FleetAgentDeploymentCustomization:   fleetAgentDeploymentCustomization,
+		AgentEnvVars:                        agentEnvVars,
```

Comment:
since this wasn't here before, wasn't it just setting agentEnvVars to nil? In other words, what's the point of setting it to an empty list of objects?